src/containers/GrepCodes/GrepCodesModal.tsx

```diff
@@ -33,15 +35,22 @@ const GrepCodesModal = ({ contentUri, onClose, locale }: Props) => {
     updateArticleAndStatus,
   } = useFetchArticleData(articleId?.toString(), locale);
 
+  const onUpdateArticle = async (updated: UpdatedDraftApiType) => {
+    const res = await updateArticle(updated);
+    const sameLength = res.grepCodes?.length === article?.grepCodes.length;
+    setChanged(!sameLength);
```

Comment:
Det hadde ikkje vore mogeleg å berre oppdatere GrepCodesForm med article for å få korrekt antall koder?

Reply:
Ja, eg berre tenkte feil. Eg berre lurte på muligheten til å oppdatere kun grep-knappen for den eine ressursen i stedet for å laste alle ressurser på nytt. Om det er vanskelig er det berre å merge denne tenker eg.